utils/src/test/java/ng/softcom/android/utils/ui/ContextUtilsTest.kt

```diff
@@ -48,4 +51,74 @@ class ContextUtilsTest {
 
         assert(actualColor == expectedColor)
     }
+
+    @Test
+    fun hasPermissions_truePositiveSingle() {
+        val app: Application = ApplicationProvider.getApplicationContext()
+
+        Shadows.shadowOf(app).grantPermissions(
+            Manifest.permission.CAMERA,
+            Manifest.permission.ACCESS_COARSE_LOCATION
+        )
+
+        val actualResult = app.hasPermissions(Manifest.permission.CAMERA)
+
+        val expectedResult = true
+
+        assert(actualResult == expectedResult)
+    }
+
+    @Test
+    fun hasPermissions_truePositiveMultiple() {
```

Comment:
Maybe the long test name notation will be easier to read..

`check that hasPermissions does stuff when stuff happens `

Reply:
i think they're more technical and precise than sentence names

https://en.wikipedia.org/wiki/False_positives_and_false_negatives